src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -47,8 +50,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
   if(userData==null||userData.getAboutMe()==null) {
 	  return;
   }
-  //String aboutMe = "This is " + user + "'s about me.";
-  
+  //String aboutMe = "This is " + user + "'s about me.\n" + userData.getAboutMe();
```

Comment:
I think you could probably delete these excess code comments and replace them with explanations of the code instead!